### PR TITLE
Replace Button test cases in favour of Buttons, extend test cases

### PR DESCRIPTION
**Pull request by lukewalczak**
Since there is no more `Button` block, I've refactored Button test cases into `Buttons` and extend them.

Update as well png related to `Buttons`

### CODE REVIEW

File: test-cases/gutenberg/buttons.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+
+--------------------------------------------------------------------------------
+
+##### TC020
+
+### Render custom background color
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
Comment (pinarol) [inline test-cases/gutenberg/buttons.md:259]:
```suggestion
-   Open gutenberg web, add `Buttons` block
```

We don't need those first steps anymore, we can just render Buttons on mobile.

File: test-cases/gutenberg/buttons.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+
+--------------------------------------------------------------------------------
+
+##### TC021
+
+### Render custom text color
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
Comment (pinarol) [inline test-cases/gutenberg/buttons.md:275]:
```suggestion
-   Open gutenberg web, add `Buttons` block
```

File: test-cases/gutenberg/buttons.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+
+--------------------------------------------------------------------------------
+
+##### TC022
+
+### Render gradient background color
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
Comment (pinarol) [inline test-cases/gutenberg/buttons.md:291]:
```suggestion
-   Open gutenberg web, add `Buttons` block
```

File: test-cases/gutenberg/buttons.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+
+--------------------------------------------------------------------------------
+
+##### TC023
+
+### Fallback to default colors in case theme colors are used
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
Comment (pinarol) [inline test-cases/gutenberg/buttons.md:307]: -   Open gutenberg web, add `Buttons` block

File: test-cases/gutenberg/buttons.md [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+
+--------------------------------------------------------------------------------
+
+##### TC024
+
+### Check if selection / caret color matches font color
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
Comment (pinarol) [inline test-cases/gutenberg/buttons.md:324]:
```suggestion
-   Open gutenberg web, add `Buttons` block
```

File: test-cases/gutenberg/buttons.md [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+-   Expect Buttons are displayed side-by-side
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Add another Button inside Buttons using inserter
+
+-   Add `Buttons` block
+-   Press inserter
```
[the remark this answers is not in the document]
Comment (lukewalczak) [reply test-cases/gutenberg/buttons.md:36]: Why do we need that step?

**Comment by lukewalczak** [comment]
I refactored test cases according to your comments and merge some test cases into one. I also removed the not about known issue with Button since with current steps it won't be visible (we are creating Buttons on web instead of mobile).

File: test-cases/gutenberg/buttons.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (pinarol) [reply test-cases/gutenberg/buttons.md:36]: Hmm I think I missed the fact that it is automatically focused on a single Button when you create Buttons.

**Pull request merged**